pandas/core/sparse/frame.py

```diff
@@ -163,7 +163,9 @@ def _init_dict(self, data, index, columns, dtype=None):
 
         # TODO: figure out how to handle this case, all nan's?
         # add in any other columns we want to have (completeness)
-        nan_arr = sp_maker(np.full(len(index), np.nan))
+        nan_arr = np.empty(len(index), dtype='float64')
```

Comment:
no, we don't use np.full anywhere in the code base at all (because its not compat with older numpy). you can add that in the numpy compat issue though (e.g. to use it once we move away from 1.7)